sources/api/static-pods/src/static_pods.rs

```diff
@@ -54,13 +55,27 @@ where
     S2: AsRef<[u8]>,
 {
     let name = name.as_ref();
-
```

Comment:
nit: this newline serves a (minor) purpose; the `as_ref` call is more like parameter management than anything to do with the function body (used when we have an AsRef parameter only used in target form, and used many times) so we typically separate it with a newline to represent that anything following is "real" work as part of the function.